apps/src/templates/instructions/TopInstructions.jsx

```diff
@@ -670,7 +672,7 @@ class TopInstructions extends Component {
     return (
       <div
         style={topInstructionsStyle}
-        className="editor-column"
+        className={!standalone && 'editor-column'}
```

Comment:
I don't feel strongly about changing this, but another option here would be to use the classnames library `!standalone && 'editor-column'` = `classnames({'editor-column': !standalone})`

Reply:
A unit test for this change would be helpful!

Reply:
👍